homeassistant/helpers/collection.py

```diff
@@ -525,6 +525,9 @@ def __init__(
         self.create_schema = create_schema
         self.update_schema = update_schema
 
+        self.remove_subscription: CALLBACK_TYPE | None = None
+        self.subscribers: set[tuple[websocket_api.ActiveConnection, int]] = set()
```

Comment:
It looks like these properties should not be accessed from outside, so we should mark them as protected.

```suggestion
        self._remove_subscription: CALLBACK_TYPE | None = None
        self._subscribers: set[tuple[websocket_api.ActiveConnection, int]] = set()
```